### DISCUSSION TITLE
Failing to render in freshly installed Windows

### DISCUSSION THREAD

**Discussion by JuanFLB** [quarto-dev/quarto-cli#854] [Q&A]
Hello,
Help needed. I just got my computer reseted (Windows). I have freshly installed all the list:

- R
- Rstudio
- Rtools
- markdown
- rmarkdown
- quarto

I am trying to render a quarto document, just default html output. and all I get is this message in Job window:
`ERROR: PermissionDenied: Access is denied. (os error 5)`
However, rmd html documents knit perfectly fine. I have googled but it seems nobody has experienced this problem before, ever.
I don´t know where to look for info or what to do....

Thanks for any help in advance!

**Comment by dragonstyle** [comment]
Could you try running a quarto command like 'quarto check' from the command prompt or power shell to see whether we can get a more helpful error message? Thank you and sorry for issue!

**Reply by JuanFLB** [reply]
This is weird, seems to not be finding R?

 Checking Quarto installation......OK
      Version: 0.9.393
      Path: C:\Program Files\Quarto\bin\
      CodePage: 1252

 Checking basic markdown render....OK

 Checking Python 3 installation....(None)
      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

 Checking R installation...........(None)
      Unable to locate an installed version of R.
      Install R from https://cloud.r-project.org/

**Reply by dragonstyle** [reply]
RStudio is likely finding R and ensuring it is available when Quarto is run from there, but no errors :(.

If you try to render a very simple `qmd` file from the command line (`quarto render test.qmd`) with no R or python, just simple markdown, do you get an error (e.g the same permission denied error with any other output)?

**Reply by JuanFLB** [reply]
Apparently it has to do with working on a network folder.
Because I am working on my University's computer, I try to have all my work documents on the network drive, so everything is backed up directly. Even trying to run default qmd file the error is thrown. This does not happen with rmd files and regular knitting of html tho, it works fine.

However, I moved to regular local folder, and having the qmd in there it can render flawlessly.

What is the difference between knitting a rmd and rendering a qmd html so it is not able to do it from a network drive?

**Reply by dragonstyle** [reply]
Quarto is doing quite a bit with a set of new internals, so there is a lot that could be in play here when compared to an rmd. If you could try running the developer version (by cloning our repo), the error messages will then contain a full stack trace which would at least point us to a line of code to start the investigation. Could you give this a try?

More info about running the developer version here:

https://github.com/quarto-dev/quarto-cli#development-version

**Comment by pjt222** [comment]
Hi, I got an comparable issue (windows) while trying to render via RStudio: 

Fehler in rmarkdown:::abs_path(input) : 
  The file 'hello.qmd' does not exist.
Ruft auf: .main -> execute -> setwd -> dirname -> <Anonymous>
Ausf�hrung angehalten

quarto check via cmd gives

[>] Checking Quarto installation......OK
      Version: 0.9.393
      Path: C:\Program Files\Quarto\bin\
      CodePage: unknown

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....(None)

      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

[>] Checking R installation...........(None)

      Unable to locate an installed version of R.
      Install R from https://cloud.r-project.org/

quarto check via cmd with administrator right gives

[>] Checking Quarto installation......OK
      Version: 0.9.393
      Path: C:\Program Files\Quarto\bin\
      CodePage: unknown

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....(None)

      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

[>] Checking R installation...........OK
      Version: 4.1.3
      Path: C:/path/to/R/R-4.1.3
      LibPaths:
        - C:/path/to/R/R-4.1.3/library
      rmarkdown: 2.13

[>] Checking Knitr engine render......OK

**Reply by pjt222** [reply]
Hi, I got to try it out a bit more. 

Using Powershell and rendering via "quarto render path/to/hello.qmd" with administrative rights everything works well and as expected 👍 The same is true for rendering via RStudio ,if I am logged in with administrative rights.

<s>but when using my usual account with lesser rights, the error still occurs. Although</s> I have checked the installations, packages, environment variables for users and system and configured them properly. The issue then still occured.

**Disabeling Quarto preview in RStudio** (Options->R Markdown->Quarto) did the trick. 👍 Now everything works like a charm!
